src/apps/backend/modules/account/account_service.py

```diff
@@ -18,6 +18,44 @@ def create_account(*, params: CreateAccountParams) -> Account:
       last_name=account_dict.get("last_name"),
       username=account_dict.get("username"),
     )
+    
+  @staticmethod
+  def get_account_by_username(*, username: str) -> Account:
```

Comment:
I'm unable to understand the schematics of these service class functions.
Why do we need to render json here? Can't we just return the `Account` ?

Reply:
I tried to follow the same structure the existing project has been using so I also returned the same way.
Should I change it to just return the `Account` ?

Reply:
Let's just send `Account` here. No need for additional transformation. You can serialize to JSON on controller level.